### PR TITLE
feat(packages): upgrade dependencies

### PR DESCRIPTION
**Pull request by bankyadam**
**tl;dr**: this PR includes a massive dependency upgrade due to security and other reasons.

## Packages upgraded:

**Core:**
- escher-auth, escher-suiteapi-js
- flat
- lodash
- moment-timezone
- node-cache
- sprintf-js
- audit-fitler

**Release:** 
- semantic-release
- pre-commit

**Lint:** 
- eslint, eslint-config-emarsys, eslint-plugin-no-only-tests, eslint-plugin-security

**Test:** 
- chai, chai-as-promised, chai-subset
- mocha, co-mocha
- nock
- sinon, sinon-chai

### All Submissions:
* [x] Have you followed our guidelines?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### CODE REVIEW

**Comment by blacksonic** [comment]
LGTM

**Comment by bankyadam** [comment]
Thanks @blacksonic
